reporters.json

```diff
@@ -1517,6 +1534,22 @@
             }
         }
     ],
+    "Colo. App.": [
+        {
+            "cite_type": "state",
+            "editions": {
+                "Colo. App.": {
+                    "end": "1980-12-31T00:00:00",
```

Comment:
These years were wrong according to the [Indigo Book][1].

[1]: https://law.resource.org/pub/us/code/blue/IndigoBook.html